src/components/TestsResults.js

```diff
@@ -1,27 +1,29 @@
 import React from 'react';
```

Comment:
Katarina's open PR is deleting this file entirely. Is this a potential merge conflict?